lib/config.js

```diff
@@ -272,6 +272,33 @@ const conf = convict({
       format: Array,
       env: 'HPKP_PIN_SHA256'
     }
+  },
+  serverCache: {
+    redis: {
+      host: {
+        default: '127.0.0.1',
+        env: 'REDIS_HOST',
```

Comment:
There are format helpers for HOST and port herE: https://github.com/mozilla/fxa-customs-server/blob/master/lib/config/config.js#L35